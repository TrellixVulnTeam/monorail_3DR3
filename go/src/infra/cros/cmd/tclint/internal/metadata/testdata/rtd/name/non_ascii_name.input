{
    "remoteTestDrivers": [
        {
            "name": "remoteTestDrivers/tauto_is_ğŸ”¥"
        }
    ]
}