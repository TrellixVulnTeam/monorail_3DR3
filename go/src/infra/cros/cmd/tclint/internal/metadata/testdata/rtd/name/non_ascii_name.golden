[
"## Errors ##",
"RemoteTestDriver 'remoteTestDrivers/tauto_is_ðŸ”¥': name must only use ASCII characters, found non-ASCII chracters in '\"remoteTestDrivers/tauto_is_\\U0001f525\"'"
]